### PR TITLE
Create LitoralDroid-california-2017.md

### PR DESCRIPTION
**Pull request by noemen**
conference-android-image-new

### CODE REVIEW

**Comment by NightlyNexus** [comment]
It's already over, right?
It won't show up in the conference list.

File: _conferences/LitoralDroid-california-2017.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: "LitoralDroid"
+website: http://www.litoraldroid.com/
+location: São Francisco, California, USA
```
Comment (stephanenicolas) [inline _conferences/LitoralDroid-california-2017.md:4]: San Francisco ?

File: _conferences/LitoralDroid-california-2017.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: "LitoralDroid"
+website: http://www.litoraldroid.com/
```
Comment (stephanenicolas) [inline _conferences/LitoralDroid-california-2017.md:3]: This conference is strange. Was it like a talk ? I can't see a schedule, and the event seems to be finished already. I don't get it.

**Comment by JakeWharton** [comment]
This site is only for tracking future conferences at this time.

**Pull request closed without merging**